src/workflows/recipes/components/ApprovalRequest.js

```diff
@@ -115,16 +119,30 @@ class ApprovalRequest extends React.PureComponent {
       extra = <Tag color="red">Rejected</Tag>;
     }
 
-    const detailSection = isPendingApproval ? (
-      <ApprovalForm
-        approvalRequest={approvalRequest}
-        isSubmitting={isSubmitting}
-        onSubmit={this.handleSubmit}
-        errors={errors}
-      />
-    ) : (
-      <ApprovalDetails request={approvalRequest} />
-    );
+    let detailSection;
+    if (isPendingApproval) {
+      // FIXME(peterbe): Replace this with something more advanced that
```

Comment:
nit: these `FIXME` comments should be `TODO`s since there is nothing broken here it just could use refinement.

Reply:
I rewrote it and pointed to the actual github issue instead.